sql/npc_list.sql

```diff
@@ -29571,6 +29571,7 @@ INSERT INTO `npc_list` VALUES (17764598,'Gate_Chocobo_Circuit','Gate: Chocobo Ci
 INSERT INTO `npc_list` VALUES (17764599,'Zahsa_Syalmhaia','Zahsa Syalmhaia',106,13.710,2.422,-83.198,1,40,40,0,0,0,0,27,0x01000C077210722072306E406E506D6100700000,32,NULL,1);
 INSERT INTO `npc_list` VALUES (17764600,'Selele','Selele',125,106.830,-5.000,-31.527,21,40,40,0,0,0,0,27,0x01000F0600100120013001400150006000700000,32,NULL,1);
 INSERT INTO `npc_list` VALUES (17764601,'Artisan_Moogle','Artisan Moogle',96,-27.508,1.519,-41.049,21,40,40,0,0,0,0,27,0x0000520000000000000000000000000000000000,32,NULL,1);
+INSERT INTO `npc_list` VALUES (17764601,'Artisan_Moogle','Artisan Moogle',96,-27.508,1.519,-41.049,21,40,40,0,0,0,0,27,0x0000520000000000000000000000000000000000,32,'SOA',1);
```

Comment:
You have actually added a second NPC here with the same ID. You either need to delete the line above, or adjust the line above and delete your new line.